### ISSUE TITLE
The playback experiences irregular network buffering, occasionally resulting in spinning/loading circles.

### ISSUE THREAD

**Issue by maojianyou** [yuliskov/SmartTube#3452]
Because accessing resources from Canada involves routing through Hong Kong and Singapore, it essentially makes a round trip. However, when I watch official YouTube videos on my computer, they also go through nodes in Hong Kong or Singapore without buffering. However, the SmartTube app on my TV experiences more buffering. Could we optimize the network, such as improving the performance of the app under slow or congested network conditions?**
![image](https://github.com/yuliskov/SmartTube/assets/55315920/b4529acb-6e52-42e0-a835-a00e177740eb)

**Comment by apwebber** [comment]
Yeah me too, I noticed this happening a lot recently. I couldn't even get 1080p to work without buffering every 2 seconds. Switch back to the actual youtube app and 4k works right away. Appreciate there are many things that could cause this but that's my experience lately.

**Comment by jj-uk** [comment]
Same here. I cannot watch ANY live streams  eg 'sky news live'.

Just everlasting buffering on all video formats.

Non-live videos are ok.

**Comment by maojianyou** [comment]
> Yeah me too, I noticed this happening a lot recently. I couldn't even get 1080p to work without buffering every 2 seconds. Switch back to the actual youtube app and 4k works right away. Appreciate there are many things that could cause this but that's my experience lately.

The latest version seems to no longer buffer.

**Comment by PylotLight** [comment]
> > Yeah me too, I noticed this happening a lot recently. I couldn't even get 1080p to work without buffering every 2 seconds. Switch back to the actual youtube app and 4k works right away. Appreciate there are many things that could cause this but that's my experience lately.
> 
> The latest version seems to no longer buffer.

Nope, 21.69 still buffers. Hasn't been a new version in 2 weeks.

**Comment by maojianyou** [comment]
> > > Yeah me too, I noticed this happening a lot recently. I couldn't even get 1080p to work without buffering every 2 seconds. Switch back to the actual youtube app and 4k works right away. Appreciate there are many things that could cause this but that's my experience lately.
> > 
> > 
> > The latest version seems to no longer buffer.
> 
> Nope, 21.69 still buffers. Hasn't been a new version in 2 weeks.
> 
> Ah I see the betas now, all g.

I'm using version 21.77, which supports CDN, but due to my network possibly not being very good, there are still some stutters when watching

**Comment by timfea926** [comment]
Same problem, can’t watch anything.
